CMake/kwiver-setup-python.cmake

```diff
@@ -204,6 +204,62 @@ mark_as_advanced(PYTHON_ABIFLAGS)
   find_package(pybind11)
 
 
+###
+# Python Dependencies
+#
+# sets the python dependencies defiened in python/requirements.txt
+# to be a custom command of the python libraries target
+# the user flag is required when not executing inside a venv, we always pass it
```

Comment:
We should probably create a virtual environment here so that the test occurs in a new environment and then remove it once the test is complete.Maybe FIXTURES_CLEANUP can be used to remove the virtualenv https://cmake.org/cmake/help/latest/prop_test/FIXTURES_CLEANUP.html

Reply:
Good point, will make the edit.